frontend/src/App.tsx

```diff
@@ -66,6 +68,12 @@ const headersData = [home, review];
 hotjar.initialize(HJID, HJSV);
 
 const App = (): ReactElement => {
+  useEffect(() => {
+    get<any>(`/getAllData`, {
+      callback: () => {},
+    });
+  }, []);
+
```

Comment:
Great job!! Searching is so much faster this way, and it will definitely reduce the load on the database